.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 scripts/api_spec.json
 .coverage
+deno.lock
```

Comment:
Sorry @uhunnyslack, could you remove this change? We'll be committing the `deno.lock` soon 😅

Reply:
@shapirone Done.. :)